dev/dune_db_409

```diff
@@ -1,29 +0,0 @@
-load_printer threads.cma
```

Comment:
Isn't this still needed for OCaml 4.09?

Reply:
Sure but this is now in `dev/dune_db` since we only need OCaml >= 4.09 now (doc is updated in `dev/doc/build-system.dune.md`).